include/ccf/tx_id.h

```diff
@@ -14,9 +14,26 @@
 
 namespace ccf
 {
+  // Transactions occur within a fixed View. Each View generally spans a range
```

Comment:
The consolidation and these comments and are great, I was always a bit confused by the respective numbers/terms. Does this mean that the documentation needs updating too?

Reply:
This is mostly a rewording of the existing comments we had in the kv, but that's a good point that these should go in the docs - I'll doxygenate and add them now.

Reply:
Awesome, many thanks!